Jenkinsfile

```diff
@@ -49,38 +50,19 @@ pipeline {
           NPM_CONFIG_CACHE = '.npm'
           NPM_CONFIG_USERCONFIG = '.npm/rc'
           SPAWN_WRAP_SHIM_ROOT = '.npm'
+          GITHUB_PACKAGES_TOKEN = credentials('github-api-token')
```

Comment:
is this still needed? I'm not sure that it ever was for `npm install`

Reply:
Good call.  I updated the name to `GITHUB_TOKEN`